index-eng.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+    <meta charset = "UTF-8">
```

Comment:
Can you remove the space beside the "=" character in the attribute
Can you use tabs for indentation.